terraform/modules/env-dashboards/auth-dashboard.tf

```diff
@@ -0,0 +1,996 @@
+      "fill": 1,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 24
+      },
+      "id": 14,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "paceLength": 10,
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [
+        {
+          "alias": "Duration_Sum",
+          "yaxis": 2
+        },
+        {
+          "alias": "Duration",
+          "yaxis": 2
+        }
+      ],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "alias": "Average",
+          "dimensions": {
+            "FunctionName": "fusillade-${var.env}"
+          },
+          "expression": "",
+          "hide": false,
+          "highResolution": false,
+          "id": "",
+          "metricName": "Duration",
+          "namespace": "AWS/Lambda",
+          "period": "",
+          "refId": "A",
+          "region": "us-east-1",
+          "returnData": false,
+          "statistics": [
+            "Average"
+          ]
+        },
+        {
+          "alias": "Maximum",
+          "dimensions": {
+            "FunctionName": "fusillade-${var.env}"
+          },
+          "expression": "",
+          "highResolution": false,
+          "id": "",
+          "metricName": "Duration",
+          "namespace": "AWS/Lambda",
+          "period": "",
+          "refId": "B",
+          "region": "us-east-1",
+          "returnData": false,
+          "statistics": [
+            "Maximum"
+          ]
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "API Lambda Duration",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "ms",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+          "format": "ms",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "aliasColors": {
+        "Errors": "#890f02",
+        "Errors_Sum": "#890f02"
+      },
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "account-cloudwatch",
```

Comment:
Template out the name of the cloudwatch datasource with `${var.aws_cloudwatch_data_source_name}` here and elsewhere